docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,462 @@
+    var lineItems = [];
+
+    for (i = 0; i < orderJson.lineItems.physicalItems.length; i++) {
+      var thisItem = orderJson.lineItems.physicalItems[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.productId,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.salePrice,
+        "value": thisItem.extendedSalePrice,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    for (i = 0; i < orderJson.lineItems.digitalItems.length; i++) {
+      var thisItem = orderJson.lineItems.digitalItems[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.productId,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.salePrice,
+        "value": thisItem.extendedSalePrice,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    for (i = 0; i < orderJson.lineItems.giftCertificates.length; i++) {
+      var thisItem = orderJson.lineItems.giftCertificates[i];
+      orderQty += thisItem.quantity;
+      lineItems.push({
+        "content_id": thisItem.type,
+        "content_name": thisItem.name,
+        "currency": orderJson.currency.code,
+        "price": thisItem.amount,
+        "value": thisItem.amount,
+        "quantity": thisItem.quantity,
+        "content_type": "product_group"
+      });
+    }
+
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('StartCheckout', {
+        "contents": lineItems,
+        "value": orderJson.orderAmount,
+        "quantity": orderQty,
+        "currency": orderJson.currency.code
+    });
+  });
+</script>
+```
+
+## Subscribe to newsletter
+The following snippet executes when a shopper successfully subscribes to a newsletter. It initiates capture of any data the sample analytics provider collects about newsletter subscription.
+
+```handlebars title="Sample Script Code Start: Subscribe to Newsletter" lineNumbers
+<script>
+  if (window.location.pathname.indexOf('/subscribe.php') === 0 && BODL.getQueryParamValue('result') === 'success') {
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('Subscribe');
+  }
+</script>
+```
+
+## View category
+The following snippet collects data about the category that's currently part of the `BODL` instance.
+
+```handlebars title="Sample Script Code Start: View Category Content" lineNumbers
+<script>
+  if (BODL.category) {
+    sampleAnalyticsProvider.instance('<%= property_id %>').track('ViewContent', {
+      contents: BODL.category.products.map((p) => ({
+        content_id: p.id,
+        content_category: BODL.category.name,
+        content_name: p.name,
+        content_type: "product_group",
+        currency: p.price.without_tax.currency,
+        price: p.price.without_tax.value,
+        value: p.price.without_tax.value,
```

Comment:
Since the `value` fields in this and the View Product example are the same as the price, can you delete those? I think it's just a bit cleaner of an example that way!